projects/core/src/datetimepicker/calendar.html

```diff
@@ -1,72 +1,41 @@
-           [class.disabled]="!_nextEnabled()" (click)="_nextClicked()"
-           aria-label="Next month">
-        <svg width="24" height="24" viewBox="0 0 24 24">
+            </div>
+            <div class="mat-datetimepicker-calendar-period-button" [@slideCalendar]="_calendarState" (@slideCalendar.done)="_calendarStateDone()">
+                <strong>{{ _monthYearLabel }}</strong>
+            </div>
+            <div class="mat-datetimepicker-calendar-next-button" [class.disabled]="!_nextEnabled()" (click)="_nextClicked()" aria-label="Next month">
+                <svg width="24" height="24" viewBox="0 0 24 24">
           <path d="M10 6L8.59 7.41 13.17 12l-4.58 4.59L10 18l6-6z"></path>
         </svg>
-      </div>
+            </div>
+        </div>
     </div>
-  </div>
-  <mat-datetimepicker-month-view *ngSwitchCase="'month'"
-                  [activeDate]="_activeDate"
-                  [type]="type"
-                  [selected]="selected"
-                  [dateFilter]="_dateFilterForViews"
-                  (selectedChange)="_dateSelected($event)"
-                  (_userSelection)="_userSelected()">
-  </mat-datetimepicker-month-view>
-  <mat-datetimepicker-year-view *ngSwitchCase="'year'"
-                 [activeDate]="_activeDate"
-                 [type]="type"
-                 [selected]="selected"
-                 [dateFilter]="_dateFilterForViews"
-                 (selectedChange)="_monthSelected($event)"
-                 (_userSelection)="_userSelected()">
-  </mat-datetimepicker-year-view>
-  <mat-datetimepicker-clock *ngSwitchDefault
-             [startView]="_clockView"
-             [interval]="timeInterval"
-             [minDate]="minDate"
-             [maxDate]="maxDate"
-             [dateFilter]="dateFilter"
-             [selected]="_activeDate"
-             (activeDateChange)="_onActiveDateChange($event)"
-             (selectedChange)="_timeSelected($event)"
-             (_userSelection)="_userSelected()">
-  </mat-datetimepicker-clock>
-</div>
+    <mat-datetimepicker-month-view *ngSwitchCase="'month'" [activeDate]="_activeDate" [type]="type" [selected]="selected" [dateFilter]="_dateFilterForViews" (selectedChange)="_dateSelected($event)" (_userSelection)="_userSelected()">
+    </mat-datetimepicker-month-view>
+    <mat-datetimepicker-year-view *ngSwitchCase="'year'" [activeDate]="_activeDate" [type]="type" [selected]="selected" [dateFilter]="_dateFilterForViews" (selectedChange)="_monthSelected($event)" (_userSelection)="_userSelected()">
+    </mat-datetimepicker-year-view>
+    <mat-datetimepicker-clock *ngSwitchDefault [twelvehour]="twelvehour" [startView]="_clockView" [interval]="timeInterval" [minDate]="minDate" [maxDate]="maxDate" [dateFilter]="dateFilter" [selected]="_activeDate" (activeDateChange)="_onActiveDateChange($event)"
+        (selectedChange)="_timeSelected($event)" (_userSelection)="_userSelected()">
+    </mat-datetimepicker-clock>
+</div>
```

Comment:
it's hard to see the actual changes if you reformat the whole file.. please do *not* do this!
line breaks were there for a reason. it's very unpleasant having to scroll to the right if you have that long lines.

Reply:
Done

Reply:
Done